app.js

```diff
@@ -1,8 +1,22 @@
-const http = require('http');
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+
+const { asyncHooks, errorHandler } = require('./middlewares');
+
 const { PORT, ENV } = require('./config');
-const { log, debug, warn, error } = require('./logger');
+const logger = require('./logger');
+
+app
+  .use(bodyParser.json())
+  .use(asyncHooks)
+  .get('/', async (req, res) => {
```

Comment:
why this is outside of the asynhooks run? in case there is an error we won't get the context